Cargo.toml

```diff
@@ -17,6 +17,7 @@ license     = "Apache-2.0/MIT"
 exclude     = ["book/*"]
```

Comment:
It was easier for me to test with it left as-is, because I could use `[patch.crates-io]` to insert it into other dependencies like `criterion-cycles-per-byte` and `pprof`.